package.json

```diff
@@ -270,7 +270,7 @@
     "vscode": "1.1.34"
   },
   "dependencies": {
-    "@cvrg-report/clover-json": "0.3.0",
+    "@cvrg-report/clover-json": "^0.3.2",
```

Comment:
can you leave this pinned to `0.3.2`?

Reply:
Sure, though I can't imagine why you want to pin all these dependencies. Bad experience with B/C breaks between minor versions?

Reply:
haha yep... normally in most software projects I let these float atleast to the minor / patch but freeze the dependencies for builds via the package-lock.json